test/unit/test_ptb.F90

```diff
@@ -215,7 +215,7 @@ subroutine test_ptb_overlap(error)
       call check_(error, ints%overlap(2, 3), overlap_exp(3), thr=thr)
       call check_(error, ints%overlap(1, 15), overlap_exp(4), thr=thr)
       call check_(error, ints%overlap(1, 23), overlap_exp(5), thr=thr)
-      call check_(error, ints%overlap(12, 22), overlap_exp(6), thr=thr)
+      call check_(error, ints%overlap(11, 22), overlap_exp(6), thr=thr)
```

Comment:
Maybe add a comment to make clear which pair of orbitals the number corresponds to or you can change to checking the complete matrix

Reply:
I agree that in the long-term, a comparison of the full matrices would be more consistent and definitely better practice. For now, I've introduced annotations on which matrix elements are actually compared.